cpp/ql/src/Security/CWE/CWE-611/XXE.ql

```diff
@@ -13,344 +13,10 @@
  */
 
 import cpp
-import semmle.code.cpp.ir.dataflow.DataFlow
+import XML
+import Xerces
+import Libxml2
```

Comment:
Maybe the two libraries should be imported right after the definition of the `XxeFlowStateTransformer` class in `XML.ql`? In that way, we're sure that no one accidentally imports `XML` without any of the implementation classes?

Reply:
I've pushed that solution, but let me know if you don't like it.

Reply:
Yeah, it should be fine to import them at the top 👍.